chainer/backends/cuda.py

```diff
@@ -36,8 +36,8 @@
 import six
 
 import chainer
-from chainer.backends import intel64
-from chainer.configuration import config
+import chainer.backends.intel64
+import chainer.configuration
```

Comment:
It's too verbose to import every module in this way. Also, it incurs extra symbol look-up cost in runtime.
Could you revert those changes?